cli/README.md

```diff
@@ -9,12 +9,12 @@ Download precompiled binaries here: <br>
 
 Install as a Go package. Requires Go 1.18 or higher. [[Download](https://golang.org/dl/)]
 ```
-go get github.com/BattlesnakeOfficial/rules/cli/battlesnake
+go install github.com/BattlesnakeOfficial/rules/cli/battlesnake@latest
 ```
 
 Compile from source. Also requires Go 1.18 or higher.
 ```
-git clone git@github.com:BattlesnakeOfficial/rules.git
+git clone https://github.com/BattlesnakeOfficial/rules.git
```

Comment:
FYI - this isn't really the "wrong" URL, it's just the one that uses your locally configured SSH credentials.

Check out this page for more info on how to set it up yourself:
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/about-ssh

However, I do think using the HTTPS URL is better for this use case, because the error you see if you don't have the key set up is confusing.

Reply:
Thanks for the helpful suggestion! I will definitely improve this.🙌